src/rovr_control/rovr_control/auto_dig_server.py

```diff
@@ -1,31 +1,26 @@
 import rclpy
+from rclpy.node import Node
 from rclpy.action import ActionServer
 
-from std_msgs.msg import Bool
-
 from rovr_interfaces.action import AutoDig
 from rovr_interfaces.srv import Drive, Stop, SetPosition, SetPower
 from rclpy.action.server import ServerGoalHandle, CancelResponse
 
-from rovr_control.node_util import AsyncNode
+from asyncio import sleep
 
 
-class AutoDigServer(AsyncNode):
+class AutoDigServer(Node):
     def __init__(self):
         super().__init__("auto_dig_server")
         self._action_server = ActionServer(
             self, AutoDig, "auto_dig", self.execute_callback, cancel_callback=self.cancel_callback
         )
 
-        # TODO: This should not be needed anymore after ticket #257 is implemented!
-        self.digger_goal_subscription = self.create_subscription(
-            Bool, "/digger/goal_reached", self.digger_goal_callback, 10
-        )
-
         self.cli_drivetrain_drive = self.create_client(Drive, "drivetrain/drive")
         self.cli_drivetrain_stop = self.create_client(Stop, "drivetrain/stop")
 
         self.cli_lift_zero = self.create_client(Stop, "lift/zero")
+        self.cli_lift_lower = self.create_client(Stop, "lift/lower")
```

Comment:
@Isopod00 Why are we using Stop Service type? Can we use something else, because this is a bit confusing

Reply:
Yes, I am going to open a PR today to fix this in general